Package.swift

```diff
@@ -10,14 +10,16 @@ let package = Package(
         .library(name: "SourceryJS", targets: ["SourceryJS"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/kylef/Commander.git", from: "0.6.1"),
+        .package(url: "https://github.com/kylef/Commander.git", .exact("0.6.1")),
         // PathKit needs to be exact to avoid a SwiftPM bug where dependency resolution takes a very long time.
         .package(url: "https://github.com/kylef/PathKit.git", .exact("0.8.0")),
-        .package(url: "https://github.com/jpsim/SourceKitten.git", from: "0.18.1"),
-        .package(url: "https://github.com/IBM-Swift/CommonCrypto.git", from: "0.1.5"),
```

Comment:
I am seeing `0.1.1` for libCommonCrypto in [Podfile.lock](https://github.com/krzysztofzablocki/Sourcery/blob/56b999afa87272bb9c70b907ff4e1922723ba91c/Podfile.lock#L4).